web/app/themes/mitlib-parent/inc/content-location-2021.php

```diff
@@ -66,16 +66,13 @@
 	$strLocation = 'noThumbs';
 }
 
-$alertTitle   = cf( 'alert_title' );
-$alertContent = cf( 'alert_content' );
+$alert_title   = cf( 'alert_title' );
```

Comment:
Can you remind me where `cf` comes from? I'm going to approve the PR but I feel like I've asked before and still can't remember.